src/packages/cli/src/bin.ts

```diff
@@ -175,9 +178,17 @@ async function main(): Promise<number> {
     return 1
   }
   console.log(result)
+
+  // Project hash is a SHA256 of the schemaPath
+  const projectHash = await getProjectHash()
+  // SHA256 of the cli path
+  const cliPathHash = await getCLIPathHash()
+
   // check prisma for updates
   const checkResult = await checkpoint.check({
     product: 'prisma',
+    cli_path_hash: cliPathHash,
+    project: projectHash,
```

Comment:
So one is named `cli_path_hash` the other `project`, why not `projectHash` or `project_path_hash`?

Reply:
Renamed to `project_hash`